all.sln

```diff
@@ -41,7 +41,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ActorClient", "samples\Acto
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AspNetCore", "AspNetCore", "{DCBEAB71-0BDE-42F4-8187-F576942CB038}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Dapr.Client.Grpc", "src\Dapr.Client.Grpc\Dapr.Client.Grpc.csproj", "{BCE5A721-69C5-4510-97ED-931ADE587466}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Dapr.Client.Autogen.Grpc", "src\Dapr.Client.Autogen.Grpc\Dapr.Client.Autogen.Grpc.csproj", "{BCE5A721-69C5-4510-97ED-931ADE587466}"
```

Comment:
What's the role of this as a separate project now that we're taking a dependency on it? Should this and `Dapr.Client` just be one project?

Reply:
The raw autogenerated client code would still be available as a separate package if users want to consume it directly.